libbpf-tools/tcprtt.c

```diff
@@ -131,14 +130,14 @@ static error_t parse_arg(int key, char *arg, struct argp_state *state)
 			fprintf(stderr, "invalid local address: %s\n", arg);
 			argp_usage(state);
 		}
-		env.laddr = htonl(addr.s_addr);
+		env.laddr = addr.s_addr;
 		break;
 	case 'A':
 		if (inet_aton(arg, &addr) < 0) {
 			fprintf(stderr, "invalid remote address: %s\n", arg);
 			argp_usage(state);
 		}
-		env.raddr = htonl(addr.s_addr);
+		env.raddr = addr.s_addr;
```

Comment:
IIRC, this single `htonl` here is to avoid multiple `bpf_ntohl` in BPF program. This change here seems to be wrong.

Reply:
You are right.

> inet_aton() converts the Internet host address cp from the IPv4 numbers-and-dots notation into binary  form  (in
       network  byte  order) and stores it in the structure that inp points to.